### PR TITLE
Added Winston library for logging

### PR DESCRIPTION
**Pull request by rakhimundhada15**
Added the Winston library to improve logging for lambda functions.

### CODE REVIEW

**Review by scott-jones-AlertLogic — APPROVED** [review]
Looks good, thank you Rakhi!

File: logger/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ */
+
+const { createLogger, format, transports } = require('winston');
+const { combine, timestamp, errors, json } = format;
+
+const logger = createLogger({
+    level: process.env.LOG_LEVEL || 'info',
+    format: combine(
+        timestamp(),
+        errors({ stack: true }),
```
Comment (kkuzmin) [inline logger/index.js:17]: I think we don't need stack for all errors. We only need them for exceptions and for errors which are handles gracefully we normally don't need a stack trace. Is is possible to pass this parameter into `logger.error()`?

File: index.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ module.exports = {
     AlAwsCollector : require('./al_aws_collector'),
     Util : require('./al_aws'),
     Health : require('./health_checks'),
-    Stats : require('./statistics_templates')
+    Stats: require('./statistics_templates'),
+    logger: require('./logger')
```
Comment (kkuzmin) [inline index.js:16]: Please keep all export names consistent - `Logger`

**Review by kkuzmin — CHANGES_REQUESTED** [review]
@rakhimundhada15 thanks. LGTM but some minor comments.

File: logger/index.js (hunk 1 above)
Comment (kkuzmin) [reply logger/index.js:17]: Let's expose error options through env var as well, like `LOG_ERROR_OPTS`

File: logger/index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ * 
+ * Logger setup for Lambda functions
+ * @end
+ * 
+ */
+
+const { createLogger, format, transports } = require('winston');
+const { combine, timestamp, errors, json } = format;
+
+const showStack = process.env.LOG_ERROR_OPTS ? { stack: process.env.LOG_ERROR_OPTS } : {};
```
Comment (kkuzmin) [inline logger/index.js:13]: I suggest we pass `LOG_ERROR_OPTS ` as is, like
```
process.env.LOG_ERROR_OPTS ? JSON.parse(process.env.LOG_ERROR_OPTS) : {};
```

**Review by kkuzmin — APPROVED** [review]

**Pull request merged**